### DISCUSSION TITLE
Dashboard is no longer reachable

### DISCUSSION THREAD

**Discussion by ethanlchristensen** [jgyates/genmon#1130] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

1.19.03

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

I am just reaching out to ask about the premade pHAT's from print size me. I got the premade kit with the raspbery pi zero.

I got it all set up and was working for some time (a week). Just recently (today), I can no longer reach the dashboard, and actually can no longer reach the pi via ssh or ping. 

I did notice one thing. When I set up the pi, the LED on the pHAT was blue. I brought the pi back inside now to investigate and the LED on the pHAT is now a solid bright green and no longer blue. Does this idicate anything important?

**Comment by skipfire** [comment]
The initial blue should have been a dim blue and should have gone away after it fully booted up.  Green, unless something else has been configured to use the LED, should indicate that the confirmation portal is active without hotspot, though you might check if the captive portal hotspot did turn on.  The green LED should only turn on after pressing the button next to it 3 times within 5 seconds.  If you reboot/power-cycle the Pi it should have the dim blue for a short period while it boots, then go dark once the software is loaded unless you use the button.

Do you have the stuff to hook up a display, keyboard, and mouse to the Pi?  If so that will be the easiest way to diagnose what's going on.

**Reply by ethanlchristensen** [reply]
I do not, I had ran into some issues with the kit, and decided to install a new raspberrypi os lite ( a week ago ). I set it up and all worked for about a week. Today I could no longer connect and the light on the pHAT is green now.

Heres what happens:
- I plug the pi into power
- The light is a dim blue
- After sometime the light turns to bright green

**Reply by skipfire** [reply]
Ok, if you reinstalled the OS then I have no clue why it is coming up green.  Did you enable any of the GPIO add-ons or install any other software on the Pi?

**Reply by ethanlchristensen** [reply]
The light never came up green before. It would just turn off after some time. All I have done on the pi is install the OS, and install genmon.

**Reply by ethanlchristensen** [reply]
Is there a certain setup you do on your end before shipping out the prebuilt kit?

**Reply by skipfire** [reply]
The setup includes log2ram, the captive portal software, and a test package; then I boot and run the Pi and run the test package that validates all the hardware functions correctly.  I'm searching to see if I can find anything that would cause the green LED to be lit (GPIO 23).

**Reply by ethanlchristensen** [reply]
I do know, based on the dashboard, the CPU got to 80+C (before it stopped working). I wonder if the SSD got to hot and started failing . . .

**Comment by skipfire** [comment]
Overheating is a possibility.  I've not found anything that would trigger GPIO 23 to be set high without some code doing it, so it may indicate something internally got stuck in the Pi.  What issues were you running into before you reformatted?  I'm wondering if there are any clues there.

**Reply by ethanlchristensen** [reply]
Hello @skipfire, meant to follow up sooner. We just rebuilt genmon on new PI and added shade to the container housing the pi outdoors and haven't had an issue since. . .  perhaps the heat played with the SSD in some way.

**Comment by Jkh62** [comment]
Right now my outside temp is 80.4F 
PI CPU is 116.7F
CPU load  14%

85C /  185F is the limit for my PI and it will throttle down if it hits that limit. 
Mine has never come close on days in the 90's .

Might want to look at your CPU load if something it kicking it up.

I put my Genmon enclosure on the back side of my gen , out of direct sun all day.


https://imgur.com/tRQV7f9
